MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1187 @@
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+             List<string> list = new List<string>()
+            {
+                "Home",
+                "Food",
+                "Utilities",
+                "Education"
+            };
+            comboBoxItem.ItemsSource = list;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+#### Enabling editing mode in combobox
+
+By default, in the combobox editor, you can select a needed item by scrolling the whole list of items.
+
+You can filter a particular item in the list items by using the `IsEditable` property, this property enables you to type a list item in the combo box editor.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+           comboBoxItem.IsEditable = true;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Searching words that starts with the input text
+
+Displays all the matches that contain first character of the typed characters in items source of combobox editor. This strategy is case in-sensitive.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+            comboBoxItem.TextSearchMode = DataFormTextSearchMode.StartsWith;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Searching words that contains the input text
+
+Displays all the matches that contain the typed characters in items source of combobox editor. This strategy is case in-sensitive.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+            comboBoxItem.TextSearchMode = DataFormTextSearchMode.Contains;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing the height of combobox item 
+
+By default the combo box drop down list item height is `400d` , You can change the size for the drop down list item size by using `MaxDropDownHeight` property in the `DataFormItem`.
```

Comment:
`DataFormItem` => `DataFormComboBoxItem`

Reply:
DataFormItem to DataFormSwitchItem changed